src/vanilla/Model/MethodTS.cs

```diff
@@ -569,39 +532,28 @@ public string GenerateWithHttpOperationResponseMethod(string emptyLine)
             return builder.ToString();
         }
 
-        public void GenerateOperationArguments(TSArgumentList operationArguments)
+        public void GenerateOperationArguments(TSObject operationArguments)
         {
-            operationArguments.Object(obj =>
+            ParameterTransformations transformations = GetParameterTransformations();
+            Action<string, string> addArgument = (string operationArgumentName, string operationArgumentValue) =>
```

Comment:
Suggest using a local function for this

Reply:
```cs
void addArgument(string operationArgumentName, string operationArgumentValue)
{
    ...
}
```

Reply:
Done in https://github.com/Azure/autorest.typescript/pull/211/commits/65d262eca39c20e4affd29ec9f696aa810c2cf41.